README.md

```diff
@@ -70,6 +70,11 @@ def test_cirq_compile():
 
 We run benchmarks regularly to compare against the most recent versions of the most popular quantum compiler frameworks for a range of circuits. Here's the latest:
 ![alt text](benchmarks/latest_compiler_benchmarks_by_circuit.png)
+
+In addition to raw compilation stats, we simulate the compiled circuits with a noisy density matrix simulation to see how each compiler impacts performance:
```

Comment:
Is it worth a sentence or two on what the absolute error definition is here? Or maybe better in the docs?

Reply:
Related+for my own understanding.

We have two circuits, the original `O` and the compiled `C = compile(O)`.  For some measurement/metric `M`, we then generate

* `M(O)`
* `M(noisy_simulation(C)) = M(noisy_simulation(compile(O))`

My questions are
1. Is the relative error we plot the ratio of the above two measurements?
2. Are we assuming `M(0)` and `M(C)` are equal? In principle yes, but are all circuit transformations lossless, especially if floating point precision comes into play?

cc @Misty-W as this also related to the benchmark specific measurement work.

Reply:
We do have approximate compilation passes in UCC, they are:
- Optimize1qGates: Approximates single-qubit unitaries up to numerical precision.
- UnitarySynthesis: Approximates arbitrary unitaries with a decomposition based on a given synthesis method.
- Collect2qBlocks: Can group two-qubit operations in ways that lead to approximate synthesis (in some cases).

Agreed, benchmarking circuits for noise resilience is a great follow-up.